src/completions/History.ts

```diff
@@ -67,6 +67,11 @@ export class HistoryCompletionSource extends Completions.CompletionSourceFuse {
                 options = args.slice(0, 2).join(" ")
                 query = args.slice(2).join(" ")
             }
+            if (query.startsWith("-b")) {
+                let args = query.split(" ")
+                options = args.slice(0, 2).join(" ")
+                query = args.slice(2).join(" ")
```

Comment:
Thank you for opening this pull request! :)

I think `-b` doesn't take any arguments (that's what the doc says here I believe: https://github.com/cmcaine/tridactyl/blob/581fda074e21ec62dd549954d54aa6fa7045a9cb/src/excmds.ts#L1506 ) so this should probably be `args.slice(0,1)` instead of `args.slice(2)`. I think you can take inspiration from what's happening with the `-private` argument for `winopen`, since it doesn't take any arguments either.
Doesn't `tabopen -b this is a test` start a search request for `is a test` with this current code?

>Eh it's also a bit ugly in case of code duplication

Don't worry about the ugliness of the code, everything will be replaced one day :).

Reply:
Thanks, your comment makes sense. Is it easy to test a version that I produced? I'll take a look later

Reply:
I wouldn't say that was so difficult:

```
git clone [your repo URL]
sudo apt-get install npm
npm install
npm run build
npm run run
```

Reply:
Thanks you two it was real easy :-)
Glacambre, I just incorporated your first suggestion changing args.slice(0,2) to args.slice(0,1). The -private argument for winopen has different code, I don't understand the difference. But this works, so I suppose I'm done. My first PR yaay